package.json

```diff
@@ -127,9 +127,9 @@
     "regenerator-runtime": "^0.13.9",
     "require-reload": "^0.2.2",
     "simple-git": "^2.47.1",
-    "styled-components": "5.3.1",
+    "styled-components": "5.3.3",
     "tarball-extract": "^0.0.6",
-    "ts-jest": "27.0.5",
+    "ts-jest": "27.1.1",
```

Comment:
Any reason not to have the carrot on these? (ex. `"styled-components": "^5.3.3"`)

Reply:
good catch I'll put it back not sure why it got taken out